event_handler/sources.py

```diff
@@ -158,13 +184,16 @@ def get_source(headers):
     "tekton": EventSource(
         "tekton-secret", simple_token_verification
         ),
+    "circleci": EventSource(
+        "Circleci-Signature", circleci_verification
+        ),
     "argocd": EventSource(
         "Argo-Signature", simple_token_verification
         ),
     "pagerduty": EventSource(
-        "X-PagerDuty-Signature", pagerduty_verification
+        "X-Pagerduty-Signature", pagerduty_verification
```

Comment:
Should this be changing? Seems to be the opposite of the above.

Reply:
Yep.

It is the opposite of the change above, but it's correct AFAIK, because python(/Flask?) is doing some odd case transformation when extracting headers (i.e. `X-PagerDuty-Signature` -> `X-Pagerduty-Signature`, even though the RFC states HTTP header names should be case-insensitive), so this matches the value extracted and passed from the header in `event_handler.py`.